### ISSUE TITLE
Use accessible submit buttons on all forms

### ISSUE THREAD

**Issue by PaulMorris** [SolutionGuidance/psm#553]
As part of Issue #510, and related to issue #65, we should use accessible submit buttons for forms.

The HTML CodeSniffer error: "This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid."

**Comment by PaulMorris** [comment]
There are two overlapping changes involved:
  - making sure each `<form>` has a `<button>` (or `<input>`) with `type="submit"`
  - converting all buttons on the site to `<button>` rather than `<a>`.

Only the first is needed for accessibility compliance, so it makes sense to start with that and then follow up with the second in a separate issue.  There are: 

37 `<form>`s.  
151 buttons total, with classes: blueBtn (10), purpleBtn (61), and greyBtn (80). 
75 buttons are `href="javascript:;"`  (easiest to convert)
34 buttons are `href="` ...a javascript function call
28 buttons are `href=${...}` with a JSP insert (22 of those are just ${ctx} to start a url)
17 buttons are `href=<c:url ...`

**Comment by jasonaowen** [comment]
The form and multiple buttons on the My Profile list (in `provider/profile/list.jsp`) are a strange alternative to using simple links. I propose we fix that page by removing the form entirely, and replacing it with links in the form of `View | Edit | Renew`. So, we'd remove the radio box table column, add a new links column, and move the view link from being the NPI (because I think some provider types can be enrolled without an NPI - like Personal Care Assistant - which leads to `<a>` links with no link text) to an explicit "View" link, and then remove the form entirely.

![myprofile](https://user-images.githubusercontent.com/1494855/33898006-0eccb56c-df35-11e7-915b-e12140cc06ea.png)

**Comment by cecilia-donnelly** [comment]
@jasonaowen I like this approach!
